mappings/net/minecraft/class_5315.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5315
+	FIELD field_25024 version I
+	FIELD field_25025 lastPlayed J
+	FIELD field_25026 versionName Ljava/lang/String;
+	FIELD field_25027 worldVersion I
+	FIELD field_25028 stable Z
+	METHOD <init> (IJLjava/lang/String;IZ)V
+		ARG 1 version
+		ARG 2 lastPlayed
+		ARG 4 versionName
+		ARG 5 worldVersion
+		ARG 6 stable
+	METHOD method_29022 getVersion ()I
+	METHOD method_29023 fromDynamic (Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/class_5315;
+	METHOD method_29024 getLastPlayed ()J
+	METHOD method_29025 getVersionName ()Ljava/lang/String;
+	METHOD method_29026 getWorldVersion ()I
```

Comment:
should we prefer "data version" or "world version" name?

Reply:
Hmm, what is it called in `MinecraftVersion`?

Reply:
`world_version` in `version.json`, but it is called `DataVersion` in like persistent data, level.dat, etc, and the community uses "data version" as a convention already. Imo the `world_version` is not accurate, as it should rather be "save version" if we follow this logic.